### ISSUE TITLE
tel URIs don't dial or display

### ISSUE THREAD

**Issue by ari** [BelledonneCommunications/linphone-desktop#513]
# How to reproduce

1. Click on link like `<a href="tel:1234">this</a>`
2. Set it to open in Linphone
3. Linphone opens, but does not display or dial the number

This was tested with Linphone Desktop 4.3.0-alpha-97-g8aa04c28 - Qt5.15.2 on OSX.

**Comment by ibroom** [comment]
Set the default telephone app in the Facetime preferences on your mac then try again.

**Comment by ari** [comment]
I don't have Facetime configured on my machine at all. But the problem isn't that Linphone doesn't launch. It launches (because the browser has it has the "tel" helper). But Linphone doesn't react and dial or show the number.

**Comment by missiskisses** [comment]
I have Facetime with default call app set to Linphone and I can confirm the issue. The app starts, but a number isn't dialed.

**Comment by awnz** [comment]
Same on Linux (me: Ubuntu 20.04). The Linphone console log shows this when I click a tel: link in the browser (phone number masked out)...

```
[09:31:26:480][0x2797770][Warning]"This command doesn't exist: `tel`."
[09:31:26:480][0x2797770][Info]"Detecting URI command: `tel:*********`..."
[09:31:26:480][0x2797770][Info]"Execute command:" QHash(("sip-address", "sip:*********"))
[09:33:01:124][0x2797770][Info]"Received command from other application: `show`."
[09:33:01:124][0x2797770][Info]"Detecting cli command: `show`..."
[09:33:01:124][0x2797770][Info]"Execute command:" QHash()
```

... but nothing happens, no call is placed, nothing happens in the UI at all. If in the background, it doesn't even come to the foreground despite the logging above around "show".

If I manually execute the Linphone AppImage with the parameter "call sip-address=*********" (including quotes) it works as intended, but this isn't achievable with the usual protocol handler registering which passes a tel: link as above.

**Comment by awnz** [comment]
Workaround (based on https://github.com/BelledonneCommunications/linphone-desktop/issues/246#issuecomment-495100602):

```
#!/bin/sh
#
# Translate the tel: link to a Linphone callto: command
number=`echo "$1" | sed 's/callto:\|tel://'`
exec ~/Applications/Linphone* "call sip-address=$number"

```
I used this in conjunction with https://github.com/TheAssassin/AppImageLauncher which placed the Linphone AppImage in the ~/Applications folder and manually pointed my Firefox to it by clicking a tel: link and going "Choose other application..." the first time.

**Comment by julonexus** [comment]
Hi
The things is that passing from µURL, it will try to call sip:123 that is not a sip address. So the call is not done.
When calling from CLI, it will try to call 123, that can be transform into a sip address. (sip:123@domain)
A fix is available on 5.0.
